vlib/v/checker/tests/any_int_float_ban_err.out

```diff
@@ -7,39 +7,26 @@ vlib/v/checker/tests/any_int_float_ban_err.vv:2:12: error: unknown type `int_lit
     1 | type Foo = int_literal | float_literal
     2 | type Fo2 = int_literal
       |            ~~~~~~~~~~~
-    3 | 
+    3 |
     4 | struct Int {
 vlib/v/checker/tests/any_int_float_ban_err.vv:5:7: error: unknown type `int_literal`
-    3 | 
+    3 |
     4 | struct Int {
     5 |     i int_literal
       |       ~~~~~~~~~~~
     6 |     f float_literal
     7 | }
-vlib/v/checker/tests/any_int_float_ban_err.vv:6:7: error: unknown type `float_literal`
-    4 | struct Int {
-    5 |     i int_literal
-    6 |     f float_literal
-      |       ~~~~~~~~~~~~~
-    7 | }
-    8 |
```

Comment:
Why are there .out changes for such a cleanup?

Imho all the tests should remain passing for such PRs, and the same set of errors should be stored in the .out files.

Reply:
I'll fix it.